server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryErrorTest.java

```diff
@@ -307,6 +328,48 @@ public void remoteDeliveryShouldRetryWhenDATA421() throws Exception {
             .body("[0].message", containsString("subject: test")));
     }
 
+    @Test
+    public void remoteDeliveryShouldNotDuplicateContentWhenSendPartial() throws Exception {
+        with()
+            .body(SINGLE_PARTIAL_RCPT_421_BEHAVIOR)
+            .put("/smtpBehaviors").prettyPeek();
+
+        messageSender.connect(LOCALHOST_IP, jamesServer.getProbe(SmtpGuiceProbe.class).getSmtpPort())
+            .sendMessage(MailImpl.builder()
+                .name("name")
+                .sender(new MailAddress(FROM))
+                .addRecipient(RECIPIENT1)
+                .addRecipient(RECIPIENT2)
+                .mimeMessage(MimeMessageUtil.mimeMessageFromString(MIME_MESSAGE))
+                .build());
+
+        awaitAtMostOneMinute.untilAsserted(() -> given(requestSpecification(), RESPONSE_SPECIFICATION)
```

Comment:
we tend to use `until` here

Reply:
I don't like `until` as when combined with rest-assured it makes things unessessarilly verbose.

Though I want my overall work to move forward hence I still contributed the fixup you ask for.

Do you think we should keep it?